src/serac/infrastructure/accelerator.hpp

```diff
@@ -33,11 +33,20 @@
 #include <cuda_runtime.h>
 #else  //__CUDACC__
 /**
- * @brief Macro that toggles between decorating a function for host and device or noop's for non-accelated builds.
+ * @brief Macro that toggles between decorating a function for host_device or noop's for non-accelated builds.
  */
 #define SERAC_HOST_DEVICE
+/**
+ * @brief Macro that toggles between decorating a function for host or noop's for non-accelated builds.
+ */
 #define SERAC_HOST
+/**
+ * @brief Macro that toggles between decorating a function for device or noop's for non-accelated builds.
+ */
```

Comment:
the original wording makes it sound like it marks the function as being a no-op, rather than the macro itself doing nothing

I'm not sure what the right wording should be, this is consistent with the earlier suggestion:

Macro that evaluates to {`__host__`, `__device__`} when compiling with nvcc and does nothing on a host compiler

Reply:
also, for 1-liner doxygen comments, could we consider doing something like

```cpp
/// @brief lorem ipsum dolor si amet
#define SERAC_HOST
```

so that a 1-line function doesn't balloon into like 5 lines of code when adding doxygen comments?

Reply:
I'm fine with this when appropriate. I honestly use the javadoc `/**` blocks because my vscode plugin can generate them automatically.